internal/test/wmcb/wmcb_test.go

```diff
@@ -293,6 +322,37 @@ func (vm *wmcbVM) runTestKubeletUninstall(t *testing.T) {
 	require.NoError(t, err, "TestKubeletUninstall failed")
 }
 
+// configureContainerd installs the containerd Windows service
+func (vm *wmcbVM) configureContainerd() error {
+	// Check if containerd is running
+	running := vm.isRunning(hybridOverlayServiceName)
```

Comment:
I am confused a bit.  Comment says check if containerd is running but we are checking hybridOverlayServiceName here.

Reply:
Thanks for the catch, must have messed this up during rebase with copy/paste